### PR TITLE
fix #1262 adding new message in Chinese

### LINKED ISSUE #1262: Adding new message in Chinese
adding new message in Chinese

### PR DESCRIPTION
**Pull request by DaZuiZui**
adding new message in Chinese

### CODE REVIEW

**Comment by dsyer** [comment]
I can only see question marks ("????" etc.) in `messages_cn.properties`, in browser and also when I checkout your branch. Did it get destroyed on the way to github, or on the way to my computer?

**Comment by dsyer** [comment]
I see the content now, thank you. Did you test it in a browser? I think the code for Chinese language is actually "zh" (not "cn"), so I don't expect it to work yet.

File: src/main/resources/messages/messages_zh.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+welcome=欢迎
+required=是必须的
+notFound=没有找到
+duplicate=已在使用中
+nonNumeric=非数字
+duplicateFormSubmission=重复表单提交
+typeMismatch.date=失效时间
```
Comment (dsyer) [inline src/main/resources/messages/messages_zh.properties:7]: Google translate suggests 失效日期 (and renders 失效时间 as "expiration time" which seems wrong)?

Comment (dsyer) [reply src/main/resources/messages/messages_zh.properties]: I see you changed it to 到期时间. Google translates that to "expire date" which also seems wrong. We want an "invalid date" message, not an "expiration" message. Is my suggestion also wrong?

**Comment by codespearhead** [comment]
@DaZuiZui Can you [link this PR](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to issue #1262?

To do this, you just need to add the following to the end of the PR description:

fix #1262

**Comment by snicoll** [comment]
Closing due to the lack of feedback. We can reopen if update is provided.

**Pull request closed without merging**